make/Main.gmk

```diff
@@ -1425,7 +1425,7 @@ ALL_TARGETS += $(addsuffix -only, $(filter-out dist-clean clean%, $(ALL_TARGETS)
 # are internal only, to support Init.gmk.
 
 print-targets:
-	  @$(ECHO) $(sort $(ALL_TARGETS))
+	  @$(ECHO) $(sort $(ALL_TARGETS)) | $(TR) ' ' '\n'
```

Comment:
If we are to do this, I would suggest reducing the amount of external commands used and keeping it to make functions, and also fixing the indentation.
```suggestion
	$(info $(subst $(SPACE),$(NEWLINE),$(sort $(ALL_TARGETS))))
```
I would also suggest applying the same change to all three print targets.